cpu/pmqa.py.data/pmqa.yaml

```diff
@@ -0,0 +1,7 @@
+#Usage: run_arg can  any of one from below list
```

Comment:
Or, you could take advantage of the multiplex, running always one test only in the code, and multiplexing the tests in the yaml file.